pom.xml

```diff
@@ -23,23 +23,20 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>com.appdynamics</groupId>
-			<artifactId>machineagent</artifactId>
-			<version>25.7</version>
-			<systemPath>${project.basedir}/../25.7/machineagent.jar</systemPath>
-			<scope>system</scope>
-		</dependency>
+            <groupId>com.appdynamics</groupId>
+            <artifactId>machine-agent</artifactId>
+            <version>3.7.11</version>
```

Comment:
does this version exists? even if it does, why are we providing older version of machine agent?